plugins/org.eclipse.elk.alg.layered/META-INF/MANIFEST.MF

```diff
@@ -37,3 +37,4 @@ Export-Package: org.eclipse.elk.alg.layered,
  org.eclipse.elk.alg.layered.p5edges;x-friends:="org.eclipse.elk.alg.layered.test",
  org.eclipse.elk.alg.layered.p5edges.orthogonal;x-friends:="org.eclipse.elk.alg.layered.test",
  org.eclipse.elk.alg.layered.p5edges.splines;x-friends:="org.eclipse.elk.alg.layered.test"
+Import-Package: org.eclipse.elk.alg.topdownpacking.options
```

Comment:
I am unsure whether this dependency can cause problems.
What speaks against moving all topdown options to elk.core?

As I see it all algorithms that need top-down layout should depend on topdownpacking, correct?

Reply:
This is an old import and can be removed